LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
Did you mean to delete the license? :man_shrugging:

Reply:
Yeah, i didn't see the point of having it at the moment :sweat_smile: